### ISSUE TITLE
Check here for my Compatibility list for DKWDRV

### ISSUE THREAD

**Issue by Ghaleon7771** [DKWDRV/DKWDRV#22]
I will close this thread as no one seems interested. guess i better stick with POPstarter...

PS2 Model: SCPH-79001
USB used: Kingston 14GB and Sandisk 16GB for all the tests in FAT32 of course.

In some of my tests i noticed that games that got stuck is because they were fragmented, some worked after defragmented while others still crashed nonetheless... This is common sense of course to always defragment the games.

I KNOW THAT XA/CDDA AUDIO NEEDS TO BE ADDED, I WILL POST THE MISSING AUDIO IN REPORT ANYWAY.

(List Updated: 30/4/2024)
So far this is all my list:

Ace Combat 3: perfect but music is not present anywhere.

Aura Battler Dunbine: it works well but as soon as you fight 2 or more times the 3D graphics become corrupted for some reason.

Alundra: works flawlessly.

Bloody Roar 2: perfect but  no music in battles and in fmvs (also some flickering in the beast bars in some stages)

Breath Of Fire IV: perfect, but missing music.

Bushido Blade 2: perfect but no music anywhere.

Crash Team Racing: perfect but missing some audios and music.
Aura Battler Dunbine: it works well but as soon as you fight 2 or more times the 3D graphics become corrupted for some reason.

Chrono Cross: works good (a bit too low on the framerate or its just how the game is?) there is music and effects everywhere, but in the status screen characters and menu flickers A LOT. I dont know if this is how the PS2 emulates this game or its a problem of DKWDRV.

Devilman: perfect. but no audio in fmvs

Digimon World 2: perfect but no music in fmvs.

Dragon Ball Z - Idainaru Dragon Ball Densetsu: perfect but no music in main menu.

DOOM: gets stuck in the first "LOADING..." screen.

Final Fantasy Tactics: perfect except no audio in fmvs.

Goujin Senki: gets stuck in the first FMV but you can skip it and it will work fine.

Gundam Battle Assault: perfect but  no music in battles

Gundam Battle Assault 2: crash after the Playstation logo.

Khamrai: plays fairly well, there is a notable drop in FPS in most areas, i dont know if this is how it behaves on a PS1.

Lunar SSSC: very SLOW fps.

Lunar 2: missing tilesets and sprites in game.

Need For Speed High Stakes: perfect

Need For Speed Porsche Unleashed: gets stuck in the EA logo, probably due to lack of XA/CDDA channels.

Real Robot Battle Line: works perfectly except no audio in the fmv

Samurai Deeper Kyo: perfect but  no music.

Street Fighter Alpha 3: perfect, missing music.

Suikoden 2: works perfectly.

TEKKEN 3: has got wrong 3D models.

Tenchu 1 and 2: works perfectly

The King Of Fighters 99: same as SFA3.

Twisted Metal 2: works perfectly, missing music.

Twisted Metal 4: same as TM2.

Twisted Metal Small Brawl: no music and fmvs get stuck. but its pefectly playable if you skip fmvs.

Vagrant Story: works perfectly.

Xenogears: perfect except no audio in fmvs.

**Comment by LeandroFerreira445** [comment]
I stopped using POPS a long time ago because most of the games I wanted didn't work or my saves got corrupted on the pendrive

**Comment by LeandroFerreira445** [comment]
I stopped using POPS a long time ago because most of the games I wanted didn't work or my saves got corrupted on the pendrive

**Comment by LeandroFerreira445** [comment]
Another thing I don't like is putting in those Hugo correction files, I think this converter for vcd is very annoying and it's also annoying

**Comment by Ghaleon7771** [comment]
> I stopped using POPS a long time ago because most of the games I wanted didn't work or my saves got corrupted on the pendrive

I stopped using POPS and DKWDRV and just sold my PS2 and got rid of the CRT TVs. I got tired of trying to find the "best" possible way to play those old games. I just found that emulation on a PC is the way to go. All these emulators on the PS2 are a total waste of time and effort.

**Comment by LeandroFerreira445** [comment]
I don't agree with your opinion brother but I respect it

**Comment by Ghaleon7771** [comment]
> I don't agree with your opinion brother but I respect it

Its ok, but you cant deny how bad the games run on either DKWDRV or POPS, if it werent for Hugopocked fixes most of the games in POPs are unplayable. DKWDRV seems to be promising but its really in a beta state and the author seems to not care too much about updating it. Its a waste of time while you can enjoy Duckstation with ehancements and sharp pixels without compatibility issues.

**Comment by LeandroFerreira445** [comment]
I still prefer DKWDRV my games here are running well I have no complaints but I respect your opinion

**Comment by Ghaleon7771** [comment]
> I still prefer DKWDRV my games here are running well I have no complaints but I respect your opinion

Why you keep saying "i respect your opinion" but give me thumbs down on my comments lol
Also there is a lot of missing music in DKWDRV, you enjoy games like that?

**Comment by LeandroFerreira445** [comment]
I don't see a problem for me, I know that in the future anonymous will improve DKWDRV as he said on github

**Comment by LeandroFerreira445** [comment]
two things I liked most about DKWDRV saving the data on the real memory card and using the original bin-cue game files

**Comment by Ghaleon7771** [comment]
> two things I liked most about DKWDRV saving the data on the real memory card and using the original bin-cue game files

Yeah using bin-cue instead of that terrible POPs system with the VCD is a improvement.

**Comment by Ghaleon7771** [comment]
Ok since i changed my mind some time ago regarding this, i will re open my list and continue updating it when i have time. 
At first i thought this DKWDRV project was going nowhere, but after i was surprised after i saw the updates.
